presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestingH2JdbcModule.java

```diff
@@ -26,7 +26,7 @@
 
 import static java.lang.String.format;
 
-class TestingH2JdbcModule
+public class TestingH2JdbcModule
```

Comment:
Why?

Reply:
Previously it was package protected so we cant access in our JmxTest or we have to move JmxTest to a different package

Reply:
Please move `JmxTest` to different package. I don't think we need separate package for single test, do we? Otherwise please squash commits.